h/static/scripts/annotator/plugin/texthighlights.coffee

```diff
@@ -91,34 +120,6 @@ class TextHighlight
       $(@_highlights).scrollintoview complete: ->
         resolve()
 
-# Public: Wraps the DOM Nodes within the provided range with a highlight
```

Comment:
> Did this change the code at all, or just move it?

It just moved it. (It's also a separate comment, for more transparency, see here: https://github.com/hypothesis/h/commit/3571742ef4c702c70a040310c436496808347be6 )